WordPress/gutenberg
# [RNMobile] Add media inserter blocks to toolbar

## What?
Adds buttons to insert media blocks (Image, Video, Gallery, Audio) from the mobile header toolbar when a block is not selected.

* https://github.com/wordpress-mobile/gutenberg-mobile/pull/5900

## Why?
Allows users to quickly add media when writing a post without opening the Inserter.

## How?
In the header toolbar component, this PR adds buttons for four media blocks, and allows the creation/insertion of each block directly from the toolbar.


## Testing Instructions
1. Open a post
2. When no block is selected, observe that media buttons are available within the header toolbar
3. Press a media insertion button
4. Expect that block type to be inserted

## Screenshots or screencast 
**Note**: The "Add Blocks" inserter button, undo/redo buttons, and media icons will be modified in subsequent PRs. 

https://github.com/WordPress/gutenberg/assets/643285/5ff20e72-9cd0-4d01-8482-3b374619e21b
